README.md

```diff
@@ -1 +1,85 @@
-# fast-formdata-stringify
\ No newline at end of file
+# fast-formdata-stringify
+
+This module aims to serialize your data into `application/x-www-form-urlencoded` encode.
+
+## Install
+
+```
+npm i --save fast-formdata-stringify
+```
+
+*NB:* this project is not ready for a production usage. Some feature misses. (Please PR!)
```

Comment:
"Some key features are missing, see LINK. If you would like to contribute and help, we welcome pull requests!